tools/concat/.shed.yml

```diff
@@ -0,0 +1,7 @@
+categories: [Assembly]
+description: concatenate
+name: concat
+owner: nml
+long_description: concat - Concatenates two files into a resulting file
```

Comment:
Change long_description to:  **concat - Concatenates (list) paired collection into a single list**